docs/backend-system/building-plugins-and-modules/08-migrating.md

```diff
@@ -255,3 +255,67 @@ backend.add(
 Checkout the [custom service implementations](https://backstage.io/docs/backend-system/building-backends/index#custom-service-implementations) documentation and also the [core service configurations](https://backstage.io/docs/backend-system/core-services/index) page in case you'd like to create your own custom mock factory for one or more services.
 
 3. Now you can finally start your plugin locally by running `yarn start` from the root folder of your plugin.
+
+## Remove support for the old backend system
```

Comment:
I think it's worth getting guidance for plugin libraries in there straight away too. Main thing there being to remove all dependencies on `backend-common`

Reply:
Updated here, is it good to go?
https://github.com/backstage/backstage/pull/26133/commits/e4fdf1f02d5fcfb82a9c687b5ed97264693646d6

Reply:
It's not super explicit, but it does cover it 😅. Let's go with this, I imagine the fact that those packages and all exports are deprecated will likely be enough of a nudge in the end too.